L1Trigger/TrackFindingTracklet/interface/IMATH_TrackletCalculator.h

```diff
@@ -0,0 +1,462 @@
+#ifndef L1Trigger_TrackFindingTracklet_interface_IMATH_TrackletCalculator_h
+#define L1Trigger_TrackFindingTracklet_interface_IMATH_TrackletCalculator_h
+
+#include "Settings.h"
+#include "imath.h"
+
+//
+// Constants used:
+//   dphisector
+//   rmaxL6
+//   zmaxD5
+//   rmaxdisk
+//   kr, kphi1, kz
+//
+//   rmean[], zmean[]
+
+namespace Trklet {
+  class Globals;
+};
+
+class IMATH_TrackletCalculator {
+public:
+  IMATH_TrackletCalculator(const Trklet::Settings* settings, imathGlobals* globals, int i1, int i2)
+      : settings_(settings), globals_(globals) {
+#ifndef CMSSW_GIT_HASH
+    edm::LogVerbatim("Tracklet") << "=============================================";
+    char s[1024];
```

Comment:
[suggestion]
consider [boost::format](https://www.boost.org/doc/libs/1_73_0/libs/format/doc/format.html) instead of `sprintf`.

if you do prefer to use a `printf`-like function, please replace `sprintf(s, ...)` with `snprintf(s, 1024, ...)` to avoid buffer overruns.

Reply:
now using snprintf(...)